### PR TITLE
[PORT] Wood Floor Tiles from TG. Resprite and new tiles!

### PR DESCRIPTION
**Pull request by neeshacark**
# Document the changes in your pull request

**I DELETED THE TESTING CATERGORY BECAUSE THIS WAS ORIGINALLY A VERY MINOR PR BUT IT IS INDEED ALL TESTED AND FUNCTIONS AS INTENDED**

Replaces our old wood floortile sprites with newer, nicer sprites from TG that match our tiles. https://github.com/tgstation/tgstation/pull/69342

Also takes their new wood tile sprites, which give some variants for mappers to use to spruce up their maps!

Crafting recipes for new wood tiles added.

Blutsu has resprited the wood stairs to fit, and lastly I've added thin trim variants for the wood sidings(sprites still from tg), and recolored them to match the new tiles.

# Why is this good for the game?
They look nice, and match our new tiles well

# Spriting
image
image
image
image

# Changelog

:cl:  PositiveEntropy, Cark, Blutsu
imageadd: new wood sprites from TG, and resprites on stairs by Blutsu
imagedel: replaces old wood tile sprites
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -233,6 +232,12 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 	new/datum/stack_recipe("wooden door", /obj/structure/mineral_door/wood, 10, time = 20, one_per_turf = TRUE, on_floor = TRUE), \
 	new/datum/stack_recipe("wooden sandals", /obj/item/clothing/shoes/sandal, 1), \
 
+	new/datum/stack_recipe_list("floor tiles", list( \
+	new/datum/stack_recipe("wood floor tile", /obj/item/stack/tile/wood, 1, 4, 20), \
+	new/datum/stack_recipe("parquet wood floor tile", /obj/item/stack/tile/wood/parquet, 1, 4, 20), \
+	new/datum/stack_recipe("large wood floor tile", /obj/item/stack/tile/wood/large, 1, 4, 20), \
+	new/datum/stack_recipe("tiled wood floor tile", /obj/item/stack/tile/wood/tile, 1, 4, 20), \
```
Comment (Moltijoe) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: please indent all these for clarity

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 2]
```diff
@@ -160,6 +160,18 @@
...
+	icon_state = "wood-parquet"
+	broken_states = list("wood-parquet-broken", "wood-parquet-broken2", "wood-parquet-broken3", "wood-parquet-broken4", "wood-parquet-broken5", "wood-parquet-broken6", "wood-parquet-broken7")
+
+/turf/open/floor/wood/tile
+	icon_state = "wood-tile"
+	broken_states = list("wood-tile-broken", "wood-tile-broken2", "wood-tile-broken3")
+
+/turf/open/floor/wood/large
+	icon_state = "wood-large"
+	broken_states = list("wood-large-broken", "wood-large-broken2", "wood-large-broken3")
```
Comment (Moltijoe) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: make sure to set floor_tile to the type of tile object used to place these, or when they're pulled up they're turn back into regular wood tiles

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 3]
```diff
@@ -160,6 +175,45 @@
...
+	icon_state = "wood-parquet-broken5"
+
+/turf/open/floor/wood/parquet/broken/six
+	icon_state = "wood-parquet-broken6"
+
+/turf/open/floor/wood/parquet/broken/seven
+	icon_state = "wood-parquet-broken7"
+
+/turf/open/floor/wood/tile/broken
+	icon_state = "wood-tile-broken"
```
Comment (Moltijoe) [inline code/game/turfs/simulated/floor/fancy_floor.dm:201]:
```suggestion
	icon_state = "wood-tile-broken"
	broken = TRUE
```

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 4]
```diff
@@ -160,6 +175,45 @@
 /turf/open/floor/wood/cold/broken/seven
 	icon_state = "wood-broken7"
 
+/turf/open/floor/wood/parquet/broken
+	icon_state = "wood-parquet-broken"
```
Comment (Moltijoe) [inline code/game/turfs/simulated/floor/fancy_floor.dm:179]:
```suggestion
	icon_state = "wood-parquet-broken"
	broken = TRUE
```

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 5]
```diff
@@ -160,6 +175,45 @@
...
+	icon_state = "wood-tile-broken"
+
+/turf/open/floor/wood/tile/broken/two
+	icon_state = "wood-tile-broken2"
+
+/turf/open/floor/wood/tile/broken/three
+	icon_state = "wood-tile-broken3"
+
+/turf/open/floor/wood/large/broken
+	icon_state = "wood-large-broken"
```
Comment (Moltijoe) [inline code/game/turfs/simulated/floor/fancy_floor.dm:211]:
```suggestion
	icon_state = "wood-large-broken"
	broken = TRUE
```

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]
code looks fine now

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**